package/tests/test_server.py

```diff
@@ -337,6 +351,8 @@ def test_pipelines_endpoint(cli_runner, client):
     # make sure only edges in the selected pipelines are returned
 
     assert data["edges"] == [
+        {"source": "6525f2e6", "target": "689862f0"},
```

Comment:
These random hex numbers confuse me a lot... not sure why the frontend wants them like this....